File: libdata/src/main/java/com/simprints/libdata/models/enums/VERIFY_GUID_EXISTS_RESULT.java
package com.simprints.libdata.models.enums;


public enum VERIFY_GUID_EXISTS_RESULT {
    GUID_FOUND("GUID_found"),
    GUID_NOT_FOUND_ONLINE("GUID_not_found_online"),
    GUID_NOT_FOUND_OFFLINE("GUID_not_found_offline"),
    GUID_NOT_FOUND_UNKNOWN("GUID_not_found_unknown_connectivity)");


    private String title;

    VERIFY_GUID_EXISTS_RESULT(String title) {
        this.title = title;
    }


    @Override
    public String toString() {
        return title;
    }
}

File: id/src/main/java/com/simprints/id/data/db/remote/enums/VERIFY_GUID_EXISTS_RESULT.kt
package com.simprints.id.data.db.remote.enums

import androidx.annotation.Keep

@Deprecated("Remove it with RTDB")
@Keep
enum class VERIFY_GUID_EXISTS_RESULT constructor(private val title: String) {
    GUID_FOUND("GUID_found"),
    GUID_NOT_FOUND_ONLINE("GUID_not_found_online"),
    GUID_NOT_FOUND_OFFLINE("GUID_not_found_offline"),
    GUID_NOT_FOUND_UNKNOWN("GUID_not_found_unknown_connectivity)");


    override fun toString(): String {
        return title
    }
}
